pkg/api/k8s/cluster_handler.go

```diff
@@ -54,8 +53,7 @@ func (s *clusterHandler) EnableObjectStore(config model.ObjectStore) error {
 		certName := fmt.Sprintf("rook-rgw-%s-cert", config.Name)
 		config.CertificateRef = certName
 
-		encodedCert := base64.StdEncoding.EncodeToString([]byte(config.Certificate))
```

Comment:
is this because kubernetes does the encoding for us already automatically?

Reply:
right, the api handles it for us